verify-local-matching-service-example.yml

```diff
@@ -0,0 +1,22 @@
+# This is an example configuration file to show how to configure
+# the application using a YAML file.
+
+# Dropwizard server connector configuration
+# see: http://www.dropwizard.io/1.1.2/docs/manual/configuration.html#servers
+server:
+  type: simple
+  applicationContextPath: /
+  adminContextPath: /admin
+  connector:
+    type: http
+    port: ${PORT:-50500}
```

Comment:
Do we have any specific port number ? I picked this randomly.

Reply:
That should be fine as a default